plugin/pkg/scheduler/algorithm/priorities/priorityutil/non_zero.go

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
I'm against introducing priorityutil package.
Can you please move this file to:
plugin/pkg/scheduler/algorithm/priorities/util.go
?